### ISSUE TITLE
Use ADAM optimzier

### ISSUE THREAD

**Issue by giakou4** [pjreddie/darknet#2551]
How can I use ADAM optimizer with betas and epsilon in YOLOv3 and YOLOv3 cfg files (learning rate and weight decay I can set)?
